libbeat/monitoring/namespace.go

```diff
@@ -1,34 +1,26 @@
 package monitoring
 
-import (
-	"sync"
-)
+import "sync"
 
-var namespaces = struct {
```

Comment:
@jsoriano This code change as I also already had it change in this PR. Let me now if my implementation also looks good to you.

Reply:
LGTM